channeldb/invoices.go

```diff
@@ -546,28 +546,42 @@ func (d *DB) LookupInvoice(paymentHash [32]byte) (Invoice, error) {
 	return invoice, nil
 }
 
+// InvoiceWithPaymentHash is used to store an invoice and its corresponding
+// payment hash. This struct is only used to store results of
+// ChannelDB.FetchAllInvoicesWithPaymentHash() call.
+type InvoiceWithPaymentHash struct {
+	Invoice     Invoice
```

Comment:
exported fields need godocs

Reply:
To me it seems this has a godoc. What am I missing?

Reply:
the struct has one, but the fields do not

Reply:
Sorry, missed that, fixed.